dataprofiler/tests/labelers/test_integration_column_name_data_labeler.py

```diff
@@ -0,0 +1,93 @@
+import os
+import unittest
+
+import numpy as np
+import pkg_resources
+
+from dataprofiler.labelers.column_name_model import ColumnNameModel
+from dataprofiler.labelers.data_labelers import BaseDataLabeler
+from dataprofiler.labelers.data_processing import (
+    ColumnNameModelPostprocessor,
+    DirectPassPreprocessor,
+)
+
+default_labeler_dir = pkg_resources.resource_filename("resources", "labelers")
+
+
+class TestColumnNameDataLabeler(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.one_data = ["ssn"]
+        cls.two_data = ["ssn", "failing_fail_fail"]
+
+        cls.parameters = {
+            "true_positive_dict": [
+                {"attribute": "ssn", "label": "ssn"},
+                {"attribute": "suffix", "label": "name"},
+                {"attribute": "my_home_address", "label": "address"},
+            ],
+            "false_positive_dict": [
+                {
+                    "attribute": "contract_number",
+                    "label": "ssn",
+                },
+                {
+                    "attribute": "role",
+                    "label": "name",
+                },
+                {
+                    "attribute": "send_address",
+                    "label": "address",
+                },
+            ],
+            "negative_threshold_config": 50,
+            "include_label": True,
+        }
+
+        cls.label_mapping = [
+            label["label"] for label in cls.parameters["true_positive_dict"]
+        ]
+
+        preprocessor = DirectPassPreprocessor()
+        model = ColumnNameModel(
+            label_mapping=cls.label_mapping, parameters=cls.parameters
+        )
+        postprocessor = ColumnNameModelPostprocessor(
+            true_positive_dict=cls.parameters["true_positive_dict"],
+            positive_threshold_config=85,
+        )
+
+        cls.data_labeler = BaseDataLabeler.load_with_components(
+            preprocessor=preprocessor, model=model, postprocessor=postprocessor
+        )
+
+    def test_default_model(self):
```

Comment:
do we also need a `load_from_library` test? ensure the resources work.

Reply:
Fair point yeah should test load from library for the model for sure

Reply:
added and actually caught a typo in the resources too -- 👍